lib/util.js

```diff
@@ -245,6 +246,9 @@ function getSystemErrorName(err) {
   return internalErrorName(err);
 }
 
+function getSystemErrorMap() {
+  return internalErrorMap();
+}
```

Comment:
Perhaps just avoid the aliasing in the require and export `getSystemErrorMap()` directly. This indirection is unnecessary.